src/bounding_hierarchy.rs

```diff
@@ -53,14 +87,48 @@ pub trait BoundingHierarchy {
     /// use bvh::bvh::BVH;
     /// use bvh::nalgebra::{Point3, Vector3};
     /// use bvh::ray::Ray;
-    /// # fn create_bvh() -> (BVH, Vec<AABB>) {
+    /// # use bvh::bounding_hierarchy::BHShape;
+    /// # pub struct UnitBox {
```

Comment:
You can't. We don't want to expose things from this lib which do not belong into the public interface.
And unitbox is very clearly just a testing thing.